### PR TITLE
refactor: referral program copy changes

### PR DESCRIPTION
**Pull request by sshanzel**
## Changes
- Copy changes from the Miro link below:

https://miro.com/app/board/uXjVNe2aNY8=/?messenger=slack&utm_source=slack&utm_medium=messenger&utm_campaign=notifications&utm_content=unfurled_chat_link&utm_term=messenger_analytics

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-1972 #done

### CODE REVIEW

File: packages/shared/src/components/referral/InviteLinkInput.tsx [hunk 1]
```diff
@@ -29,20 +36,25 @@ export function InviteLinkInput({
...
     }
   };
 
   const renderText = () => {
-    const copy = 'Copy link';
+    const copy = text?.initial ?? 'Copy link';
+
+    if (text?.copying) {
+      return copying ? text?.copying : copy;
+    }
```
Comment (rebelchris) [inline packages/shared/src/components/referral/InviteLinkInput.tsx:51]: Where did this come from, I don't see this part in the miro

**Review by rebelchris — CHANGES_REQUESTED** [review]
Just one I don't see

File: packages/shared/src/components/referral/InviteLinkInput.tsx (hunk 1 above)
Comment (sshanzel) [reply packages/shared/src/components/referral/InviteLinkInput.tsx:51]: When click a "Copied" button, nothing really changed, kind of a bad UX, but I can remove this if we don't want it.

Comment (rebelchris) [reply packages/shared/src/components/referral/InviteLinkInput.tsx:51]: Rather check this UX with Sab, I think they are not aware of how it works maybe.

Comment (rebelchris) [reply packages/shared/src/components/referral/InviteLinkInput.tsx:51]: Or they intended to have:
Copy link -> click -> Copied -> return back to Copy link 
As it is in current behaviour

Comment (sshanzel) [reply packages/shared/src/components/referral/InviteLinkInput.tsx:51]: Ah, I see now why there is a short delay with returning it back. In that case, the previous one was correct.

Comment (sshanzel) [reply packages/shared/src/components/referral/InviteLinkInput.tsx]: Fixed with the copied state now 🚀

**Review by rebelchris — APPROVED** [review]

**Pull request merged**